README.md

```diff
@@ -11,14 +11,33 @@ If you work a with a git rebase flow, git machete will (vastly!) help you manage
 
 ## Install
 
-Run the following commands to install git machete:
```

Comment:
Actually while you're here... pls add `[![TravisCI build status](https://api.travis-ci.org/VirtusLab/git-machete.svg?branch=master)](https://travis-ci.org/VirtusLab/git-machete)` [![TravisCI build status](https://api.travis-ci.org/VirtusLab/git-machete.svg?branch=master)](https://travis-ci.org/VirtusLab/git-machete) right after the gitter badge and before the log

Reply:
OK

Reply:
FYI, Travis is running, build should commence once you push changes https://travis-ci.org/VirtusLab/git-machete

Reply:
Nice, thank you!